### DISCUSSION TITLE
Border effect

### DISCUSSION THREAD

**Discussion by nthysgt** [ImageMagick/ImageMagick#7702] [Help] [answered]
### ImageMagick version

7.x latest

### Operating system, version and so on

linux

### Description

I have a photo without a border, and I want to create a new photo with a border like image bellow. 
Can you help me?
Thanks a lot for your assistance!"
![image](https://github.com/user-attachments/assets/c0deb6af-be95-4c3e-ab3e-d48c7c857373)

**Comment by snibgo** [comment]
Your question is vague. What is the photo without the border? What elements are used to make the border?

**Reply by nthysgt** [reply]
sorry, Because of inconvenience, I just want to create a border with a neon effect, similar to the image I provided. I'm also not sure how to describe it exactly inconvenience, I just want to create a border with a neon effect, similar to the image I provided. I'm also not sure how to describe it exactly. 
Thanks for your help

**Comment by fmw42** [comment, marked as answer]
From what I see you need two images. The background image is for the border and is larger than the foreground image.

`magick background.suffix foreground.suffix -gravity center -compose over -composite result.suffix`

If the background is not larger than the foreground, you can resize the background to be larger by 110% as follows

`magick \( foreground.suffix -set option:wd "%w" -set option:ht "%h" \) \( background.suffix -resize "%[fx:1.1*wd]x%[fx:1.1*ht]" \) +swap -gravity center -compose over -composite result.suffix`

Change the 110% as desired to make a larger border.

**Reply by nthysgt** [reply]
I will try.
Thanks for your help.

**Reply by fmw42** [reply]
Does the border come from the original (one only) image?
